inc/namespace.php

```diff
@@ -70,9 +77,22 @@ function register_settings_ui(): void {
 		SETTINGS_PAGE,
 		SETTINGS_SECTION,
 		[
+			'description' => __( 'URL of the WordPress site to use as media source.', 'amf-wordpress' ),
 			'label_for' => URL_SETTING,
 		]
 	);
+
+	add_settings_field(
+		TOKEN_SETTING,
+		__( 'Application Password', 'amf-wordpress' ),
+		__NAMESPACE__ . '\\render_field_ui',
```

Comment:
Can we use a dedicated function to render this field? We might want to rename the existing one, now that we have more than just the URL to fill in.